scripts/filevaultstatus

```diff
@@ -214,6 +214,37 @@ def get_filevault_active():
         return True
     else:
         return False
+
+def get_bootstraptoken_supported():
+    
+    cmd = ['/usr/bin/profiles', 'status', '-type', 'bootstraptoken']
+    proc = subprocess.Popen(cmd, shell=False, bufsize=-1,
+                            stdin=subprocess.PIPE,
+                            stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    (output, unused_error) = proc.communicate()
+    output = output.splitlines()
+
+    for line in output:
+        if "supported on server: YES" in line:
+            return {'bootstraptoken_supported': 1}
+    #If supported is not YES, return false
+    return {'bootstraptoken_supported': 0}
+
+
+def get_bootstraptoken_escrowed():
```

Comment:
These two functions should be combined to prevent running the command more than once